src/main/java/com/airlines/dao/CrewDaoImp.java

```diff
@@ -0,0 +1,101 @@
+package com.airlines.dao;
+
+import com.airlines.entity.Citizenship;
+import com.airlines.entity.Crew;
+import com.airlines.entity.CrewMember;
+import com.airlines.entity.Position;
+import com.airlines.exception.DaoOperationException;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CrewDaoImp implements CrewDao {
+    private static final String INSERT_CREWS_CREW_MEMBERS = "INSERT INTO crews_crew_members (crews_id, crew_members_id) VALUES(?, ?);";
+    private static final String SELECT_CREW_MEMBERS_BY_CREW__ID =
+    "SELECT * FROM crew_members  " +
```

Comment:
before commit you should make CTRL+ALT+L for autoformatting.

It's standard flow before commit and push your changes:
1. Review your classes, look for mistakes, try find it
2. Make sure code is formatted - press CTRL+ALT+L
3. Check imports, so no wildcard .* and no unused imports

Unformatted code could be easily spotted:
```
    "dsfdffff      "
         "dfdfd 
     "dfdf"
```
  as you can  all row assymetrical
  
  and formatted code:
```
      "aaaa"
      "bbbb"
      "cccc"
```
      That's formatted lines

Reply:
fixed in ed36dce27a52f51b2246b22e6eb254afa9277071